modules/ROOT/pages/as2/AS2-connector-examples-6-0.adoc

```diff
@@ -0,0 +1,90 @@
+= AS2 - Examples - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+== Example: Configure a Send AS2 Message
+
+. Place the connector in your flow as applicable.
+. To send an AS2 message with request for MDN in your Mule application, create a global `AS2 Connector Send config` element for use by the AS2 send operations.
+The `AS2 Connector Send config`  provides the following global configuration parameters:
++
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 Partner Name| Sender’s AS2 system identifier. Used for AS2-From and AS2-To headers.
+|Self config - x509 alias| Sender’s certificate and/or private key alias in the keystore. Required if using signing or encryption.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver’s AS2 system identifier. Used for AS2-From and AS2-To headers.
```

Comment:
Same as above, AS2 Partner Name is the field name as displayed in anypoint studio. as2Name is the field name as per a data weave transformation. In this case im not sure where AS2 ID has come from?

Reply:
If you don't know, then should we remove it?

Reply:
Anypoint Studio is showing **Partner config - AS2 Partner Name**. Therefore it is best to change this accordingly.